src/main/java/com/checkmarx/flow/service/JiraService.java

```diff
@@ -658,6 +659,20 @@ private void mapCustomFields(ScanRequest request, ScanResults.XIssue issue, Issu
                             log.debug("cascading values {}", value);
                             addCascadingSelect(issueBuilder, f, customField, value);
                             break;
+                        case "single-version-picker":
+                            log.debug("single version picker");
+                            issueBuilder.setFieldValue(customField, ComplexIssueInputFieldValue.with(NAME_FIELD_TYPE, value));
+                            break;
+                        case "multi-version-picker":
+                            log.debug("multi version picker");
+                            String[] selectedVersions = StringUtils.split(value, ",");
+                            List<ComplexIssueInputFieldValue> versionList = new ArrayList<>();
+                            for (String version : selectedVersions) {
+                                ComplexIssueInputFieldValue fieldValue = ComplexIssueInputFieldValue.with(NAME_FIELD_TYPE, version.trim());
+                                versionList.add(fieldValue);
+                            }
```

Comment:
You can also use something like
```
List<ComplexIssueInputFieldValue> versionList = Arrays.stream(StringUtils.split(value, ","))
        .map(version -> ComplexIssueInputFieldValue.with(NAME_FIELD_TYPE, version.trim()))
        .collect(Collectors.toList());
```

Reply:
Alright !. For now, I just made use of the existing approach.